python/pysmurf/client/command/smurf_atca_monitor.py

```diff
@@ -17,81 +17,125 @@
+            val, **kwargs)
 
-    def get_board_temp_fpga(self, slot_number=None, atca_epics_root=None, **kwargs):
+    _board_temp_fpga_reg = 'BoardTemp:FPGA'
 
+    def get_board_temp_fpga(
+            self, slot_number=None, atca_epics_root=None, **kwargs):
         """
         """
         if slot_number is None:
             slot_number=self.slot_number
         if atca_epics_root is None:
             shelf_manager=self.shelf_manager
-        return self._caget(f'{shelf_manager}:Crate:Slot_{slot_number}:' +
-                           self._board_temp_fpga,**kwargs)
+        return self._caget(
+            f'{shelf_manager}:Crate:Slot_{slot_number}:' +
+            self._board_temp_fpga_reg,**kwargs)
 
-    _board_temp_rtm = 'BoardTemp:RTM'
-
-    def get_board_temp_rtm(self, slot_number=None, atca_epics_root=None, **kwargs):
+    _board_temp_rtm_reg = 'BoardTemp:RTM'
 
+    def get_board_temp_rtm(
+            self, slot_number=None, atca_epics_root=None, **kwargs):
         """
         """
         if slot_number is None:
             slot_number=self.slot_number
         if atca_epics_root is None:
             shelf_manager=self.shelf_manager
-        return self._caget(f'{shelf_manager}:Crate:Slot_{slot_number}:' +
-                           self._board_temp_rtm,**kwargs)
-
-    _board_temp_amc0 = 'BoardTemp:AMC0'
+        return self._caget(
+            f'{shelf_manager}:Crate:Slot_{slot_number}:' +
+            self._board_temp_rtm_reg,**kwargs)
 
-    def get_board_temp_amc0(self, slot_number=None, atca_epics_root=None, **kwargs):
+    _board_temp_amc0_reg = 'BoardTemp:AMC0'
 
+    def get_board_temp_amc0(
+            self, slot_number=None, atca_epics_root=None, **kwargs):
         """
         """
         if slot_number is None:
             slot_number=self.slot_number
         if atca_epics_root is None:
             shelf_manager=self.shelf_manager
-        return self._caget(f'{shelf_manager}:Crate:Slot_{slot_number}:' +
-                           self._board_temp_amc0,**kwargs)
+        return self._caget(
+            f'{shelf_manager}:Crate:Slot_{slot_number}:' +
+            self._board_temp_amc0_reg,**kwargs)
 
-    _board_temp_amc2 = 'BoardTemp:AMC2'
+    _board_temp_amc2_reg = 'BoardTemp:AMC2'
+
+    def get_board_temp_amc2(
+            self, slot_number=None, atca_epics_root=None, **kwargs):
+        """
+        """
+        if slot_number is None:
+            slot_number=self.slot_number
+        if atca_epics_root is None:
+            shelf_manager=self.shelf_manager
+        return self._caget(
+            f'{shelf_manager}:Crate:Slot_{slot_number}:' +
+            self._board_temp_amc2_reg,**kwargs)
 
-    def get_board_temp_amc2(self, slot_number=None, atca_epics_root=None, **kwargs):
+    _junction_temp_fpga_reg = 'JunctionTemp:FPG'
 
+    def get_junction_temp_fpga(
+            self, slot_number=None, atca_epics_root=None, **kwargs):
         """
```

Comment:
doc string. specifically, we need more words on what "junction" refers to here.

Reply:
This function not a part of this PR, but agree it needs a docstring.  Created issue to address https://github.com/slaclab/pysmurf/issues/497.